### PR TITLE
(re-4387) Update ruby for CVE-2015-1855

### PR DESCRIPTION
**Pull request by shrug**

### CODE REVIEW

File: ruby/lib/ruby/1.9.1/openssl/ssl-internal.rb [hunk 1]
```diff
@@ -1,226 +1,280 @@
...
-        @svr.close
-      end
-    end
-  end
-end
+=begin
+= $RCSfile$ -- Ruby-space definitions that completes C-space funcs for SSL
+
+= Info
+  'OpenSSL for Ruby 2' project
```
Comment (melissa) [inline ruby/lib/ruby/1.9.1/openssl/ssl-internal.rb]: This is for Ruby 1.9...

**Comment by melissa** [comment]
:+1:

File: ruby/lib/ruby/1.9.1/openssl/ssl-internal.rb (hunk 1 above)
Comment (shrug) [inline ruby/lib/ruby/1.9.1/openssl/ssl-internal.rb]: This is also present in the previous version, this is a whitespace change (line endings, I imagine)

Comment (melissa) [inline ruby/lib/ruby/1.9.1/openssl/ssl-internal.rb]: Yeah, I noticed that after I wrote this. Not something we should worry about :)

**Pull request merged**